ras_backstage/resources/collection_exercise/single_collection_exercise.py

```diff
@@ -33,10 +33,14 @@ def get(short_name, period):
 
         collection_instruments = get_collection_instruments_by_classifier(survey['id'], exercise['id'])
 
+        summary_id = collection_exercise_controller.get_linked_sample_summary_id(exercise['id'])
+        sample_summary = sample_controller.get_sample_summary(summary_id)
```

Comment:
sample_controller.get_sample_summary(summary_id) if summary_id else None 
or any other indicator to the ui that there is no sample_summary